snapcraft/models/project.py

```diff
@@ -1699,30 +1652,28 @@ class Project(models.Project):
     )
     """The attributes to pass to the snap's metadata file.
 
-    These attributes are passed to ``snap.yaml`` without validation from Snapcraft.
-    This is useful for early testing of a new feature in snapd that isn't yet supported
-    by Snapcraft.
+    These attributes are passed to the ``snap.yaml`` file without validation from
+    Snapcraft. This is useful for early testing of a new feature in snapd that isn't yet
+    supported by Snapcraft.
 
-    To pass a value for a particular app, see the ``passthrough`` key for ``apps``.
+    To pass a value for a particular app, see the
+    :ref:`snapcraft-yaml-app-passthrough` key for ``apps``.
```

Comment:
```suggestion
    :ref:`snapcraft-yaml-app-passthrough` key.
```

The link text already resolves to `app.<app-name>...`